### ISSUE TITLE
Release 7.1.0 returns 404's on all endpoints

### ISSUE THREAD

**Issue by lodener** [bosh-prometheus/firehose_exporter#89]
After updating the firehose_exporter from 7.0.0 to 7.1.0, all calls to it seemingly return a 404. The process seems to properly start and listens on the configured port (`FIREHOSE_EXPORTER_LOG_LEVEL=debug`, though no additional logging seems to be shown):

```
time="2023-09-14T09:23:06Z" level=info msg="Starting firehose_exporter(version=7.1.0, branch=HEAD, revision=0f4caf3a54c63949dadf0cbe25d6ad4d8e9c9c9f)"
time="2023-09-14T09:23:06Z" level=info msg="Build context(go=go1.21.0, platform=linux/amd64, user=github-actions, date=2023-09-06T09:31:46Z, tags=unknown)"
time="2023-09-14T09:23:06Z" level=info msg="Listening on :9186"

# netstat -pant|grep firehose
tcp        0      0 0.0.0.0:9186            0.0.0.0:*               LISTEN      525336/firehose_exp
tcp        0      0 **firehose_ip**:48560       **log_api_ip**:8082        ESTABLISHED 525336/firehose_exp
```
A curl to / on the machine from which it runs (same response when calling the default `/metrics` endpoint):
```
# curl -v http://127.0.0.1:9186
*   Trying 127.0.0.1:9186...
* Connected to 127.0.0.1 (127.0.0.1) port 9186 (#0)
> GET / HTTP/1.1
> Host: 127.0.0.1:9186
> User-Agent: curl/7.81.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Content-Type: text/plain; charset=utf-8
< X-Content-Type-Options: nosniff
< Date: Thu, 14 Sep 2023 09:36:13 GMT
< Content-Length: 19
<
404 page not found
* Connection #0 to host 127.0.0.1 left intact
```

When the 7.0.0 binary is dropped in place (same VM, same config, only binary replaced), all calls are returning proper responses again:
```
time="2023-09-14T09:45:07Z" level=info msg="Starting firehose_exporter(version=7.0.0, branch=HEAD, revision=4acd22628586782ba774899bdf898a77a163a451)"
time="2023-09-14T09:45:07Z" level=info msg="Build context(go=go1.20.3, platform=linux/amd64, user=github-actions, date=2023-04-25T06:40:07Z, tags=unknown)"
time="2023-09-14T09:45:07Z" level=info msg="Listening on :9186"

# curl -v http://127.0.0.1:9186
*   Trying 127.0.0.1:9186...
* Connected to 127.0.0.1 (127.0.0.1) port 9186 (#0)
> GET / HTTP/1.1
> Host: 127.0.0.1:9186
> User-Agent: curl/7.81.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Thu, 14 Sep 2023 10:16:07 GMT
< Content-Length: 275
< Content-Type: text/html; charset=utf-8
<
<html>
				             <head><title>Cloud Foundry Firehose Exporter</title></head>
				             <body>
				             <h1>Cloud Foundry Firehose Exporter</h1>
				             <p><a href='/metrics'>Metrics</a></p>
				             </body>
* Connection #0 to host 127.0.0.1 left intact
				             </html>```

**Comment by benjaminguttmann-avtq** [comment]
Hi @lodener,

thank you for the issue. I fell over it as well and created a fix already. Currently waiting for it to get approved to make it available via the prometheus-boshrelease

**Comment by benjaminguttmann-avtq** [comment]
Hi @lodener 
issue should be fixed with version v7.1.1 which is also included in the latest prometheus bosh-release

**Comment by lodener** [comment]
Thanks @benjaminguttmann-avtq, 7.1.1 solved the issue.
